interview-questions.md

```diff
@@ -7,62 +7,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
These interview questions are an opportunity to speak on your experience (even if it is in class) with each topic. You are welcome to google an answer. You just have to paraphrase what you read. w3schools is one of my favorite places to research web development terms.